ext/exiv2/extconf.rb

```diff
@@ -1,5 +1,6 @@
 require 'mkmf'
 
+CONFIG['CXX'] = ENV['CXX'] || 'clang++'
```

Comment:
This should be solved on the user's system. I'd rather we did a check for CXX and fail loudly up front if it's not present.

Reply:
We can do that with `cxx.have_devel?` it seems: https://github.com/ruby/ruby/blob/cbf9c088f8005a49b6aa3f475c70041357774c61/lib/mkmf.rb#L3022C9-L3022C20